src/elements/clock/clock.ts

```diff
@@ -234,19 +233,35 @@ export class SbbClockElement extends LitElement {
   private async _stopClock(): Promise<void> {
     clearInterval(this._handMovement);
 
-    if (this.now) {
-      this._setHandsStartingPosition();
-    } else {
-      this._removeSecondsAnimationStyles();
-      this._removeHoursAnimationStyles();
-    }
+    this._removeSecondsAnimationStyles();
+    this._removeHoursAnimationStyles();
 
     this._clockHandHours?.removeEventListener('animationend', this._moveHoursHandFn);
     this._clockHandSeconds?.removeEventListener('animationend', this._moveMinutesHandFn);
 
     this._clockHandMinutes?.classList.add('sbb-clock__hand-minutes--no-transition');
 
     this.style?.setProperty('--sbb-clock-animation-play-state', 'paused');
+
+    if (this.now) {
+      this._resetSecondsHandAnimation();
+      this._setHandsStartingPosition();
+    }
```

Comment:
I would remove these lines from `_stopClock`, move them to a new method and call it inside the `willUpdate` (after line 96) and the `firstUpdated` (after line 296).
The reason is that they are not connected anymore to the stopping of the clock, they depend only from the `now`